gpu-operator/release-notes.rst

```diff
@@ -34,6 +34,113 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
 ----
 
+.. _v24.6.0:
+
+24.6.0
+======
+
+.. _v24.6.0-new-features:
+
+New Features
+------------
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.90.07.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit v1.16.1
+    - NVIDIA Driver Manager for Kubernetes v0.6.10
+    - NVIDIA Kubernetes Device Plugin v0.16.1
+    - NVIDIA DCGM Exporter v3.3.7-3.5.0
+    - NVIDIA DCGM v3.3.7-1
+    - Node Feature Discovery v0.16.3
+    - NVIDIA GPU Feature Discovery for Kubernetes v0.16.1
+    - NVIDIA MIG Manager for Kubernetes v0.8.0
+    - NVIDIA KubeVirt GPU Device Plugin v1.2.9
+    - NVIDIA vGPU Device Manager v0.2.7
+    - NVIDIA GDS Driver v2.17.5
+    - NVIDIA Kata Manager for Kubernetes v0.2.1
+    - NVIDIA GDRCopy Driver v2.4.1-1
+
+* Added support for NVIDIA Network Operator v24.4.0.
+  Refer to :ref:`Support for GPUDirect RDMA` and :ref:`Support for GPUDirect Storage`.
+
+* Added support for precompiled driver containers with Open Kernel module drivers.
+  Specify ``--set driver.useOpenKernelModules=true --set driver.usePrecompiled=true --set driver.version=<driver-branch>``
+  when you install or upgrade the Operator.
+  Support remains limited to Ubuntu 22.04.
+  Refer to :doc:`precompiled-drivers` for more information.
+
+  NVIDIA began publishing driver containers with this support on July 15, 2024.
+  The tags for the first containers with this support are as follows:
+
+  * <driver-branch>-5.15.0-116-generic-ubuntu22.04
+  * <driver-branch>-5.15.0-1060-nvidia-ubuntu22.04
+  * <driver-branch>-5.15.0-1063-oracle-ubuntu22.04
+  * <driver-branch>-5.15.0-1068-azure-ubuntu22.04
+  * <driver-branch>-5.15.0-1065-aws-ubuntu22.04
+
+  Precompiled driver containers built after July 15 include support for the Open Kernel module drivers.
+
+* Added support for new MIG profiles.
+
+  * For H200 devices:
+
+    * ``1g.18gb``
+    * ``1g.18gb+me``
+    * ``1g.35gb``
+    * ``2g.35gb``
+    * ``3g.71gb``
+    * ``4g.71gb``
+    * ``7g.141gb``
+
+  * Added an ``all-balanced`` profile for H20 devices that creates the following GPU instances:
+
+    * ``1g.12gb`` :math:`\times` 2
+    * ``2g.24gb`` :math:`\times` 1
+    * ``3g.48gb`` :math:`\times` 1
+
+.. _v24.6.0-fixed-issues:
+
+Fixed Issues
+------------
+
+* Role-based access controls for the following components were reviewed and revised to use least-required privileges:
+
+  * GPU Operator
```

Comment:
RBAC cleanup was also performed for the following components:

- GPU Feature Discovery
- K8s Device Plugin
- KubeVirt Device Plugin
- vGPU Host Manager